packages/components/collapse/src/collapse-item.vue

```diff
@@ -49,7 +49,7 @@
 </template>
 <script lang="ts">
 import { defineComponent, PropType, inject, computed, ref } from 'vue'
-import { CollapseProvider } from './collapse'
+import type { CollapseProvider } from './collapse.type'
```

Comment:
Could you please move this line 52 to 55?

Reply:
Yep, makes sense to move `import type` after the imports 👍 . Done :)